js/manager.js

```diff
@@ -11,6 +11,14 @@ const globalTargetingArguments = {};
 const DFPManager = Object.assign(new EventEmitter().setMaxListeners(0), {
   setTargetingArguments(data) {
     Object.assign(globalTargetingArguments, data);
+    this.getGoogletag().then((googletag) => {
+      googletag.cmd.push(() => {
+        pubadsService = googletag.pubads();
```

Comment:
can u please declare a local var:
``` const pubadsService = googletag.pubads(); ```
i know, out of this pr's scope :-), can u please make the same change to the line *42* and  delete the variable declared in line 7 ?.